deployments/a11y/image/postBuild

```diff
@@ -5,3 +5,35 @@ set -eux
 
 npm install -g pa11y-ci@3.1.0
 npm install -g pa11y@6.2.3
+
+# Install Node.js and Yarn for DH-299
+echo "Installing Node.js and Yarn..."
+apt-get update && apt-get install -y nodejs npm
```

Comment:
I don't think this can run as root. You might want to install nodejs, npm, yarn, etc. through conda.

It'd probably be quicker to nail down these steps in a live environment on the hub rather than iterating through CI.

Reply:
Thanks @ryanlovett! Will do